fiftyone/plugins/__init__.py

```diff
@@ -1,45 +1,28 @@
 """
+FiftyOne plugins.
+
 | Copyright 2017-2023, Voxel51, Inc.
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
 import types
-import deprecated
-import wrapt
 
 from .core import (
     enable_plugin,
     disable_plugin,
     delete_plugin,
+    list_plugins,
     list_disabled_plugins,
     list_downloaded_plugins,
     list_enabled_plugins,
     find_plugin,
     download_plugin,
     create_plugin,
 )
-from .definitions import list_plugins
-
-
-@deprecated.deprecated(reason="Use list_plugins() instead")
-def list_all_plugins():
```

Comment:
I included this because @ritch had a plugin that used it. Also ritchie mentioned that there are some users that have already been developing plugins, and just in case they used this method, I didn't want their plugin to break...